shotover-proxy/src/protocols/cassandra_protocol2.rs

```diff
@@ -201,8 +201,8 @@ impl CassandraCodec2 {
             Value::NULL => SQLValue::Null,
             Value::Bytes(b) => SQLValue::SingleQuotedString(String::from_utf8(b.to_vec()).unwrap()), // todo: this is definitely wrong
             Value::Strings(s) => SQLValue::SingleQuotedString(s.clone()),
-            Value::Integer(i) => SQLValue::Number(i.to_string()),
-            Value::Float(f) => SQLValue::Number(f.to_string()),
+            Value::Integer(i) => SQLValue::Number(i.to_string(), false),
```

Comment:
What on earth does this bool field mean?
Its not documented at all https://docs.rs/sqlparser/0.12.0/sqlparser/ast/enum.Value.html

Reply:
![image](https://user-images.githubusercontent.com/5120858/140009591-7d51d249-a29c-4fbb-a484-a90a1f93bb46.png)
If its true its a long.

Reply:
From a quick search I dont think cassandra differentiates between long int and regular int literals.
So always using false should be fine.

Reply:
Yeah, should've explained that. It was previously just default false so I left it at that.